dojo/filters.py

```diff
@@ -1543,18 +1546,21 @@ def filter(self, qs, value):
 
 class FindingFilterHelper(FilterSet):
     title = CharFilter(lookup_expr="icontains")
-    date = DateRangeFilter(field_name="date", label="Date Discovered")
-    on = DateFilter(field_name="date", lookup_expr="exact", label="On")
-    before = DateFilter(field_name="date", lookup_expr="lt", label="Before")
-    after = DateFilter(field_name="date", lookup_expr="gt", label="After")
+    date = DateFromToRangeFilter(field_name="date", label="Date Discovered")
```

Comment:
```suggestion
    date = DateRangeFilter(field_name="date", label="Date Discovered")
```
This change was reverted back to `DateRangeFilter` in the #10760 to accommodate date changes that are relative (i.e. X days ago) to today